### PR TITLE
16 app design line chart and pie chart

### LINKED ISSUE #16: App design: Line chart and Pie chart
Implement the line chart that shows the time series of the crime rate and the pie chart that shows the crime rate percentage by neighborhood

### PR DESCRIPTION
**Pull request by s-voon**
Hi guys, I added the time series for our dashboard. Please review it and let me know what you think!

### CODE REVIEW

**Review by MoNorouzi23 — APPROVED** [review]
Thanks Sharon;

File: src/app.py [hunk 1]
```diff
@@ -74,6 +93,32 @@
...
 ])
 
+# Define callback to update line chart based on dropdown selection
+@app.callback(
+    Output('crime-line-chart', 'figure'),
+    [Input('crime-type-dropdown', 'value'),
+     Input('neighbourhood-dropdown','value')]
+)
+def update_line_chart(selected_crime,selected_neighbourhood):
+
```
Comment (786213750) [inline src/app.py]: Do we need docstring?

File: src/app.py [hunk 2]
```diff
@@ -74,6 +93,32 @@
...
+    # Hourly data by Crime Type
+    hourly_df = crime_df.groupby(['TYPE','NEIGHBOURHOOD']).resample('h').size().reset_index(name='COUNT')
```
Comment (786213750) [inline src/app.py]: Maybe move this to the front so it don't rerun in every update

**Review by 786213750 — APPROVED** [review]
Look great! Just small changes.

File: src/app.py (hunk 1 above)
Comment (s-voon) [reply src/app.py]: I dont remember it being a hard requirement. Do you think we can implement it next milesstone?

**Pull request merged**